src/main/java/uk/gov/pay/api/service/ConnectorUriGenerator.java

```diff
@@ -85,11 +85,9 @@ public String eventsURI(Account account, Optional<ZonedDateTime> toDate, Optiona
 
         Map<String, String> params = new LinkedHashMap<>();
 
-        if (toDate.isPresent())
-            params.put("to_date", DateTimeUtils.toUTCDateString(toDate.get()));
+        toDate.ifPresent(zonedDateTime -> params.put("to_date", DateTimeUtils.toUTCDateString(zonedDateTime)));
```

Comment:
I think we can just use the formatter directly here:

```java
toDate.ifPresent(zonedDateTime -> params.put(ISO_INSTANT_MILLISECOND_PRECISION.format(zonedDateTime));
```

Reply:
That's well spotted. We can remove this wrapper method and use the formatter directly :)